apps/portal/app/buckets/[address]/_components/metadata-panel.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * A component to display object metadata in a more compact format.
+ * Designed to be used alongside the file preview area.
+ */
+import { ChevronDown, ChevronUp } from "lucide-react";
+import { useState } from "react";
+
+import CollapsedStringDisplay from "@recallnet/ui/recall/collapsed-string-display";
+
+import Metric from "@/components/metric";
+import { arrayToDisplay } from "@/lib/convert-matadata";
+
+interface MetadataPanelProps {
+  object: {
+    blobHash: string;
+    recoveryHash: string;
+    size: bigint;
+    expiry: bigint;
+    metadata: readonly {
+      key: string;
+      value: string;
+    }[];
+  };
+  objectSize: {
+    val: string | number;
+    unit: string | undefined;
+    formatted?: string;
+  };
+  objectExpiryDisplay?: string;
+  objectExpiryIso?: string;
+  objectBlockDiff?: bigint;
+}
+
+export function MetadataPanel({
+  object,
+  objectSize,
+  objectExpiryDisplay,
+  objectExpiryIso,
+  objectBlockDiff,
+}: MetadataPanelProps) {
+  const [expanded, setExpanded] = useState(false);
+
+  // Convert number val to string if needed
+  const sizeVal =
+    typeof objectSize.val === "number"
+      ? objectSize.val.toString()
+      : objectSize.val;
+
+  // Ensure unit is not undefined
+  const sizeUnit = objectSize.unit || "";
+
+  return (
+    <div className="border-t">
+      <div
+        className="hover:bg-muted/50 flex cursor-pointer items-center justify-between p-3"
+        onClick={() => setExpanded(!expanded)}
+      >
+        <h3 className="text-sm font-medium">Object Metadata</h3>
+        {expanded ? <ChevronUp size={16} /> : <ChevronDown size={16} />}
+      </div>
+
+      {expanded && (
+        <div className="space-y-4 p-4">
+          <div className="grid grid-cols-2 gap-4">
```

Comment:
This grid was previously responsive so it would be a single column by default, then use two cols for larger screen sizes.